rtp/src/rtcp/mod.rs

```diff
@@ -202,6 +206,7 @@ impl Rtcp {
         match self {
             Rtcp::SenderReport(v) => v.reports.is_full(),
             Rtcp::ReceiverReport(v) => v.reports.is_full(),
+            Rtcp::ExtendedReport(_) => false,
```

Comment:
That's a strange value.

This function is used to determine if the RTCP packets to be sent can be combined into a larger UDP packet with all RTCP reports stapled together. By saying `false` here, you effectively say that it can be combined forever together.

The only good choices are _something that checks if there is more space_, or `true` if it's impossible to know, or that the report definitely must be sent by itself.

Reply:
Right I see what you mean. The buffer capacity is checked external from this mechanic, so `false` could work _IF_ we are allowed to merge multiple XR. Confusingly TWCC could in theory be merged, but Chrome does't accept it, so the question is whether XR would be accepted in a merged form. `true` means we don't merge.